app/Models/lists.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class lists extends Model
```

Comment:
models should be named in singular and capitalized. pls also rename the file
```suggestion
class List extends Model
```